roadmap/implementers-guide/src/runtime/disputes2.md

```diff
@@ -0,0 +1,54 @@
+# Disputes
+
+## Motivation
+
+If the transaction can be verified the defined verification code, it is good. If not, there is a dispute
+about a block, which made it through the backing stage, but was then during secondary checking found
+to be invalid.
+At this point the validator set can not be trusted (since they voted for the block despite something being
+fishy at the very least) or the validator that blew the whistle has ulterior motives to do so.
+In either way, there are malicious validators around.
+As a consequence, all validators at the time of block backing, are being notified via broadcast of the first challenging vote.
+Validators that backed the candidate implicitly voted for backing the candidate and may not vote a second time.
+That would be equal to an attempted double vote and would be slashed accordingly.
```

Comment:
It's optional if we count their second vote, but we cannot permit later votes to replace their earlier votes of any voting type (dispute, approval, or backing), so it's almost surely simplest to allow voters to disagree with themselves.

Reply:
But why would we allow a voter to contradict themselves? Isn't that already bad behaviour? If we would accept multiple votes of a validator, that would influence their weight. The simplest approach would be, to allow exactly one kind of vote per validator, and silently ignore vote types that are on the same side per validator (multiple plus votes are ok, but only the first one is accounted for), voting in differently with different vote types should be an attempted double vote.

Reply:
We cannot ignore votes if we cannot know which vote came first.  As I said below, we'll just count all votes because this harms the guy who votes multiple ways even without special handling.

Reply:
How do we deal with multple votes that are both pro (backing + explicit pro vote), where one comes during the open dispute, but the second comes in after dispute resolution. Then we would need to either reward the full or the reduced amount, imho it's the earlier.